Formula/mongosh.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Mongosh < Formula
+  desc "MongoDB Shell to connect, configure, query, and work with your MongoDB database"
+  homepage "https://github.com/mongodb-js/mongosh#readme"
+  url "https://registry.npmjs.org/@mongosh/cli-repl/-/cli-repl-0.14.0.tgz"
+  sha256 "7b5a140689b4460a8b87008e6b7e7cb19acbc6e6cd1ab713e1a8923f3a995ca8"
+  license "Apache-2.0"
+
+  depends_on "node@14"
```

Comment:
Does it not work with modern nodeJS versions?

Reply:
Good point - unfortunately right now `mongosh` transitively depends on some packages that require native addons and for those there are no pre-built binaries available (only for v14). Right now that's `mongodb-client-encryption` and Kerberos support will also be added soon that suffers from the same issue. The work to support more recent Node versions is on the radar for the respective teams but unfortunately not available at the moment.